ospd/misc.py

```diff
@@ -196,7 +213,7 @@ def del_results_for_stopped_hosts(self, scan_id: str) -> None:
         """ Remove results from the result table for those host
         """
         unfinished_hosts = self.get_hosts_unfinished(scan_id)
-        for result in self.results_iterator(scan_id, False):
+        for result in self.results_iterator(scan_id, False, 0):
```

Comment:
Using keyword makes this code more readable.
```suggestion
        for result in self.results_iterator(scan_id, pop_res=False, max_res=0):
```

Reply:
With my suggestion from about it would be just
```
for result in self.results_iterator(scan_id, pop_res=False)
```
or
```
for result in self.results_iterator(scan_id)
```